spec/controllers/users_controller_spec.rb

```diff
@@ -30,6 +30,20 @@
     end
   end
 
+  describe "#me" do
+    let(:user) { valid_user }
+    context "when it is a valid request", valid_request: true do
+      it "gets single user information" do
+        get :me, id: user.id, format: :json
```

Comment:
that's not right! `me` route should be on the `collection`, not `member`. in `config/routes.rb`, before your `resources :user`, place `get 'users/me', to: 'users#me'` and remove the current `get :me` on the member.